yarpl/include/yarpl/observable/Observer.h

```diff
@@ -13,7 +13,7 @@ namespace yarpl {
 namespace observable {
 
 template <typename T>
-class Observer : public virtual Refcounted {
+class Observer : public virtual Refcounted, public yarpl::enable_get_ref {
```

Comment:
curious.. inheriting from yarpl::enable_get_ref doesn't have to be public, right? its just for the class internal stuff..

Reply:
It unfortunately does, or the compiler complains about illegally casting to the base class.